openhtf/plugs/__init__.py

```diff
@@ -481,6 +481,10 @@ def initialize_plugs(self, plug_types=None):
       self.update_plug(plug_type, plug_instance)
     self._create_or_update_rpc_server()
 
+  def get_plug(self, plug_name):
```

Comment:
`get_plug_by_class_path` or something, since `plug_name` isn't 'user', it's 'openhtf.plugs.user_input.UserInput'

Also, this deserves a bit more of a docstring, with at least an example of what this is and what it's meant for